scss/jquery/overrides/_ui-dialog.scss

```diff
@@ -3,3 +3,23 @@
 .ui-dialog {
   @include ui-dialog;
 }
+
+#{$outside-namespace-marker} {
+  .crm-container.ui-dialog {
+    &.fullscreen {
+      bottom: 0;
+      max-height: none !important;
+
+      .ui-dialog-buttonpane {
+        bottom: 0;
+        left: 0;
+        position: absolute !important;
```

Comment:
Please add a reason to why we are adding `!important` tag. Can't it be solved by w/o using `!important` tag?

Reply:
This is actually not required, thanks for flagging. Removed.